ProductionSystem/Service/ProductionManagerHandler.py

```diff
@@ -7,8 +7,8 @@
 from DIRAC.Core.DISET.RequestHandler import RequestHandler
 from DIRAC.ProductionSystem.DB.ProductionDB import ProductionDB
 
-prodTypes = [basestring, int]
-transTypes = [basestring, int, list]
+prodTypes = [basestring, int, long]
```

Comment:
Both `basestring` and `long` are python2 only things. You should use `six.string_types` instead of `basestring`and `six.integer_types` instead of  `int, long`.

Reply:
These changes can then be extended to all the code of course.